### PR TITLE
Main Pages (Landing, Submit Snap, Curation, Marketplace)

### PR DESCRIPTION
**Pull request by JulianNeuss**
Develop the front-end for the main pages of Decentraland Snap:
- Landing
- Submit Snap
- Curation
- Marketplace

### CODE REVIEW

**Comment by yemel** [comment]
I'm getting some errors while building it.

```
[FRONT]  ERROR #98124  WEBPACK
[FRONT]
[FRONT] Generating development JavaScript bundle failed
[FRONT]
[FRONT] Can't resolve '../../public/MarketplaceBackground.jpg' in
[FRONT] '/Users/yemel/dcl/snap/src/pages'
[FRONT]
[FRONT] If you're trying to use a package make sure that
[FRONT] '../../public/MarketplaceBackground.jpg' is installed. If you're trying to use a
[FRONT]  local file make sure that the path is correct.
[FRONT]
[FRONT] File: src/pages/marketplace.module.css
[FRONT]

[FRONT]  ERROR #98124  WEBPACK
[FRONT]
[FRONT] Generating development JavaScript bundle failed
[FRONT]
[FRONT] Can't resolve '../../public/submitSnapBack.jpg' in
[FRONT] '/Users/yemel/dcl/snap/src/pages'
[FRONT]
[FRONT] If you're trying to use a package make sure that
[FRONT] '../../public/submitSnapBack.jpg' is installed. If you're trying to use a local
[FRONT] file make sure that the path is correct.
[FRONT]
[FRONT] File: src/pages/curation.module.css
[FRONT]

[FRONT]  ERROR #98124  WEBPACK
[FRONT]
[FRONT] Generating development JavaScript bundle failed
[FRONT]
[FRONT] Can't resolve '../../public/submitSnapBack.jpg' in
[FRONT] '/Users/yemel/dcl/snap/src/pages'
[FRONT]
[FRONT] If you're trying to use a package make sure that
[FRONT] '../../public/submitSnapBack.jpg' is installed. If you're trying to use a local
[FRONT] file make sure that the path is correct.
[FRONT]
[FRONT] File: src/pages/submitSnap.module.css
[FRONT]

[FRONT]  ERROR #98124  WEBPACK
[FRONT]
[FRONT] Generating development JavaScript bundle failed
[FRONT]
[FRONT] Can't resolve '@material-ui/core/utils' in
[FRONT] '/Users/yemel/dcl/snap/node_modules/@material-ui/icons/utils'
[FRONT]
[FRONT] If you're trying to use a package make sure that '@material-ui/core/utils' is
[FRONT] installed. If you're trying to use a local file make sure that the path is
[FRONT] correct.
[FRONT]
[FRONT] File: node_modules/@material-ui/icons/utils/createSvgIcon.js
[FRONT]

[FRONT]  ERROR #98124  WEBPACK
[FRONT]
[FRONT] Generating development JavaScript bundle failed
[FRONT]
[FRONT] Can't resolve '@mui/material/utils' in
[FRONT] '/Users/yemel/dcl/snap/node_modules/@mui/icons-material/utils'
[FRONT]
[FRONT] If you're trying to use a package make sure that '@mui/material/utils' is
[FRONT] installed. If you're trying to use a local file make sure that the path is
[FRONT] correct.
[FRONT]
[FRONT] File: node_modules/@mui/icons-material/utils/createSvgIcon.js
[FRONT]
failed Building development bundle - 30.485s
```

**Comment by JulianNeuss** [comment]
You are missing 2 things. 

1) Install new libraries --> npm install

2) The images I forgot to upload =)

**Comment by yemel** [comment]
Just one error left:

```
[GATSBY]  ERROR #98124  WEBPACK
[GATSBY]
[GATSBY] Generating JavaScript bundles failed
[GATSBY]
[GATSBY] Can't resolve '@mui/material/utils' in
[GATSBY] '/Users/yemel/dcl/snap/node_modules/@mui/icons-material/utils'
[GATSBY]
[GATSBY] If you're trying to use a package make sure that '@mui/material/utils' is
[GATSBY] installed. If you're trying to use a local file make sure that the path is
[GATSBY] correct.
[GATSBY]
[GATSBY] File: node_modules/@mui/icons-material/utils/createSvgIcon.js
```

File: src/pages/landing.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        image={"Snap.png"}
+        imageLocation={"right"}
+        hasButton={"no"}
+        title={"Decentraland Snap"}
+        desc={
+          "Flooding the Internet with Epic Memories. An experiment in play-2-earn mechanics"
+        }
+      />
+      <ImageWithDescription
+        image={"Camera.png"}
```
Comment (yemel) [inline src/pages/landing.tsx]: /Camera.png

**Comment by yemel** [comment]
LGTM

**Pull request merged**